src/style.css

```diff
@@ -1,3 +1,67 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+
+  --blue: #00acdc;
```

Comment:
Variable name should describe the purpose, for example, if this active/hover color changes to red you have to change it in all places

Reply:
I see, thanks!